src/index.njk

```diff
@@ -10,7 +10,8 @@ tags: index-page # Note: This adds a custom class to layout.njk
     <div class="row">
       <div class="col-lg-6 d-flex flex-column justify-content-center pt-4 pt-lg-0 order-2 order-lg-1" data-aos="fade-up" data-aos-delay="200">
         <h1>Gateway Policies for Kubernetes</h1>
-        <h2>Gateways play a pivotal role in application connectivity. With Kuadrant, platform engineers and application developers can easily connect, secure and protect their services and infrastructure using its powerful policy APIs.</h2>
+        <h2>Application connectivity requirements go beyond routing traffic to the correct backend service.  You need to manage how that traffic reaches your ingress gateways, ensure secure communications, control access and protect backend services. Gateway API provides the traffic routing API. Kuadrant extends Gateway API to add a connectivity management API that makes it easy for platform engineers and application developers to collaborate, define and enforce a full connectivity solution.
```

Comment:
in two minds about the word full as we don't really do anything on the E/W side of things yet

Reply:
I think the blurb might be too long for where it's supposed to go<img width="1562" alt="Screenshot 2025-03-12 at 13 48 11" src="https://github.com/user-attachments/assets/82b2bc5c-102d-4581-9064-eaca1cd5bb17" />

Maybe only use part of it here?
<img width="1105" alt="Screenshot 2025-03-12 at 13 49 36" src="https://github.com/user-attachments/assets/dc845086-2c9d-41d7-9f60-1ee6a430a3c6" />

Reply:
Yes this makes sense thanks

Reply:
Done !